test/powershell/Modules/Microsoft.PowerShell.Utility/Sort-Object.Tests.ps1

```diff
@@ -222,3 +222,229 @@ Describe "Sort-Object DRT Unit Tests" -Tags "CI" {
 		$results[2] | Should Be 1
 	}
 }
+
+Describe 'Sort-Object Top and Bottom Unit Tests' -Tags 'CI' {
+
+	# Helper function to compare two sort entries
+	function Compare-SortEntry
+	{
+		param($nSortEntry, $fullSortEntry)
+		if ($nSortEntry -is [System.Array]) {
+			# Arrays must be compared differently than everything else
+			$nSortEntry.Equals($fullSortEntry) | Should Be $true
```

Comment:
I'll add the comment and update to the other reference equality test you suggested. I use reference equality because I'm verifying arrays are sorted into the proper position.